src/test/suite/index.ts

```diff
@@ -10,7 +10,7 @@ export const run = (): Promise<void> => {
     const mocha = new Mocha({
         ui: `tdd`,
         color: true,
-        timeout: 10000
+        timeout: 20000
```

Comment:
Any reason to increase the timeout for mocha ?

Reply:
Found a smaller file and reverted the timeout!